Gems/Atom/Asset/Shader/Code/Source/Editor/ShaderVariantAssetBuilder.cpp

```diff
@@ -1140,10 +1141,137 @@ namespace AZ
                 }
             }
 
+            if (shaderVariantInfo.m_enableRegisterAnalysis && strcmp(creationContext.m_shaderPlatformInterface.GetAPIName().GetCStr(), "vulkan") == 0)
+            {
+                AZ::IO::FixedMaxPath projectBuildPath = AZ::Utils::GetExecutableDirectory();
+                projectBuildPath = projectBuildPath.RemoveFilename(); // profile
+                projectBuildPath = projectBuildPath.RemoveFilename(); // bin
+
+                AZStd::string spirvPath;
+                AZStd::string spirvFile = AZStd::string::format(
+                    "%s_vulkan_%u.spirv.bin", creationContext.m_shaderStemNamePrefix.c_str(), shaderVariantInfo.m_stableId);
+                AZ::StringFunc::Path::Join(creationContext.m_tempDirPath.c_str(), spirvFile.c_str(), spirvPath, true, true);
+
+                AZStd::string rgaCommand = AZStd::string::format(
+                    "-s vk-spv-offline --isa ./disassem_%u.txt --livereg ./livereg_%u.txt --asic %s",
+                    shaderVariantInfo.m_stableId,
+                    shaderVariantInfo.m_stableId,
+                    shaderVariantInfo.m_asic.c_str());
+
+                AZStd::string command = AZStd::string::format("%s%s %s %s", projectBuildPath.c_str(), RgaExePath, rgaCommand.c_str(), spirvPath.c_str());
+                AZ_TracePrintf(ShaderVariantAssetBuilderName, "Rga command %s", command.c_str());
```

Comment:
FYI `AZ_TracePrintf doesn't automatically add a new line to the end of the message so the next log is slammed directly next to the previous one.
```suggestion
                AZ_TracePrintf(ShaderVariantAssetBuilderName, "Rga command %s\n", command.c_str());
```

Reply:
Thanks for the info!